### PR TITLE
Add delete_custom_resource example

### PR DESCRIPTION
**Pull request by djkonro**

### CODE REVIEW

File: examples/delete_custom_resource.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+
+from pprint import pprint
+from time import sleep
```
Comment (pokoli) [inline examples/delete_custom_resource.py]: I don't see sleep is used, so it can be removed.

**Comment by mbohlool** [comment]
@djkonro friendly ping

**Comment by djkonro** [comment]
@mbohlool , I have updated the code

**Comment by scottilee** [comment]
@djkonro do you want to update this example or close it for now since it seems to be out-of-date. For example, `read_third_party_resource` is not an available method anymore.

**Pull request closed without merging**